### DISCUSSION TITLE
Invalid class error message | tried resetting the wmi repos

### DISCUSSION THREAD

**Discussion by andreitomacnm** [SafeExamBrowser/seb-win-refactoring#383] [Q&A]
I have a student that has come to me saying that they got an error message saying invalid class.
![image](https://user-images.githubusercontent.com/40607163/159311785-3d32abe6-c227-4e44-8f41-c5552285cb6e.png)

I have gotten them to reset the WMI repos and that did not work, any suggestions? Their exam is at the end of the week and they don't really have access to another computer, all I can think of is to reset the device and see if that works, however that seems overkill and might not even work.

**Comment by dbuechel** [comment]
Which SEB version are they running?

**Reply by andreitomacnm** [reply]
the latest version for windows

**Reply by dbuechel** [reply]
Could you please specify the exact version number and potentially provide the log files of the affected session?

**Reply by andreitomacnm** [reply]
The version of SEB is 3.3.2, the student didn't even get a chance to get a log file as that error was the first thing that came up - is there a way of producing / finding the log files?
